Source/WebCore/rendering/mac/RenderThemeMac.mm

```diff
@@ -1474,28 +1474,14 @@ static void paintAttachmentIcon(const RenderAttachment& attachment, GraphicsCont
 
 static std::pair<RefPtr<Image>, float> createAttachmentPlaceholderImage(float deviceScaleFactor, const AttachmentLayout& layout)
 {
-#if HAVE(ALTERNATE_ICONS)
-ALLOW_DEPRECATED_DECLARATIONS_BEGIN
-    auto image = [NSImage _imageWithSystemSymbolName:@"arrow.down.circle"];
-ALLOW_DEPRECATED_DECLARATIONS_END
+    RetainPtr configuration = [NSImageSymbolConfiguration configurationWithPointSize:32 weight:NSFontWeightRegular scale:NSImageSymbolScaleMedium];
+    RetainPtr image = [[NSImage imageWithSystemSymbolName:@"arrow.down.circle" accessibilityDescription:nil] imageWithSymbolConfiguration:configuration.get()];
     auto imageSize = FloatSize([image size]);
     auto imageSizeScales = deviceScaleFactor * layout.iconRect.size() / imageSize;
     imageSize.scale(std::min(imageSizeScales.width(), imageSizeScales.height()));
     auto imageRect = NSMakeRect(0, 0, imageSize.width(), imageSize.height());
-    auto cgImage = [image CGImageForProposedRect:&imageRect context:nil hints:@{
-ALLOW_DEPRECATED_DECLARATIONS_BEGIN
-        NSImageHintSymbolFont : [NSFont systemFontOfSize:32],
-        NSImageHintSymbolScale : @(NSImageSymbolScaleMedium)
-ALLOW_DEPRECATED_DECLARATIONS_END
-    }];
+    auto cgImage = [image CGImageForProposedRect:&imageRect context:nil hints:nil];
```

Comment:
```suggestion
    RetainPtr cgImage = [image CGImageForProposedRect:&imageRect context:nil hints:nil];
```

Reply:
heh, I caught the other two but not this one. good call.